l: 0.4572
m: 13.5
Ixx: 0.16
Iyy: 0.16
Izz: 0.16
d_lin: [-4.03, -6.22, -5.18, -0.07, -0.07, -0.07]
d_quad: [-18.18, -21.66, -36.99, -1.55, -1.55, -1.55]
W: 112.80
B: 114.80
cog: [0., 0., 0.02]
cob: [0., 0., 0.]
neutral_bouy: False

tcm: [[0.707,  0.707,   0.707,  0.707,      0.0,     0.0,      0.0,     0.0],
      [-0.707, 0.707,  0.707,  -0.707,      0.0,     0.0,      0.0,     0.0],
      [      0.0,     0.0,     0.0,    0.0,      -1.0,     1.0,      1.0,     -1.0],
      [      0.0,     0.0,     0.0,    0.0, -0.21805, -0.21805, 0.21805, 0.21805],
      [      0.0,     0.0,     0.0,    0.0,    -0.12,   0.12,     -0.12,    0.12],
      [-0.1888, 0.1888, -0.1888, 0.1888,      0.0,     0.0,      0.0 ,    0.0]]

# tcm: [[0.707, 0.707, -0.707, -0.707, 0.0, 0.0, 0.0, 0.0],
#       [-0.707, 0.707, -0.707, 0.707, 0.0, 0.0, 0.0, 0.0],
#       [0.0, 0.0, 0.0, 0.0, -1.0, 1.0, 1.0, -1.0],
#       [0.06, -0.06, 0.06, -0.06, -0.218, -0.218, 0.218, 0.218],
#       [0.06, 0.06, -0.06, -0.06, 0.120, -0.120, 0.120, -0.120],
#       [-0.1888, 0.1888, 0.1888, -0.1888, 0.0, 0.0, 0.0, 0.0]]

m_added: [-5.50, -12.70, -14.60, -0.12, -0.12, -0.12]

